CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
 #### [v0.5.2](https://github.com/acifani/soccer-go/compare/v0.5.0...v0.5.2)
 
+> 03 November 2019
```

Comment:
this will be automatically generated while publishing a new version, so please remove it 😄

Reply:
As this is my first contribution, I was unsure if that was the case. Either way, it is fixed now :P